doc/user/content/sql/create-sink.md

```diff
@@ -68,6 +68,8 @@ Field                | Value type | Description
 `acks`               | `text`     | Sets the number of Kafka replicas that must acknowledge Materialize writes. Accepts values [-1,1000]. `-1` (the default) specifies all replicas.
 `retention_ms`       | `long`     | Sets the maximum time Kafka will retain a log.  Accepts values [-1, ...]. `-1` specifics no time limit.  If not set, uses the broker default.
 `retention_bytes`    | `long`     | Sets the maximum size a Kafka partion can grow before removing old logs.  Accepts values [-1, ...]. `-1` specifics no size limit.  If not set, uses the broker default.
+`avro_key_fullname`  | `text`     | Sets the Avro fullname on the generated key schema, if a `KEY` is specified. When used, a value must be specified for `avro_value_fullname`. The default fullname is `row`.
+`avro_value_fullname`| `text      | Sets the Avro fullname on the generated value schema. The default fullname is `envelope`.
```

Comment:
```suggestion
`avro_value_fullname`| `text`      | Sets the Avro fullname on the generated value schema. The default fullname is `envelope`.
```

Reply:
Also we should probably mention that value fullname requires key fullname to be specified (if a key exists).